MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,387 @@
+
+{% tabs %}
+{% highlight C# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### DataForm display options attribute
+
+The order of the editors in the group can also be changed by using `RowOrder` property of [DataFormDisplayOptions] attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+public class ContactInfo
+{
+    [Display(GroupName = "Details")]
+    [DataFormDisplayOptions(RowOrder = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Details")]
+    [DataFormDisplayOptions(RowOrder = 2)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Details")]
+    [DataFormDisplayOptions(RowOrder = 1)]
+    public string MiddleName { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The order of the editors in the group can also be changed by handling the `GenerateDataFormItem` event using `RowOrder` property of a `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 0;
+            }
+            else if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.RowOrder = 2;
+            }
+            else if (e.DataFormItem.FieldName == "MiddleName")
+            {
+                e.DataFormItem.RowOrder = 1;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editors order of the group in grid rows
+
+The order of the editors of group in grid row can be changed within a row by using `RowOrder` and `ItemsOrderInRow` properties[DataFormDisplayOptions] attribute and by handling `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 2)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 1)]
+    public string MiddleName { get; set; }
+
+{% endhighlight %}
+{% highlight C# %}
+
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.GroupName == "Name")
+            {
+                e.DataFormGroupItem.ColumnCount = 3;
+            }
+        }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Changing group name
```

Comment:
## Changing the group name for the editors

Reply:
changed heading